packages/commonwealth/client/scripts/helpers/index.ts

```diff
@@ -410,3 +410,20 @@ export function baseToNetwork(n: ChainBase): ChainNetwork {
       return null;
   }
 }
+
+// Decimals For Tokens
+export function getDecimals(): number {
```

Comment:
let's pass in an IChainAdapter (eg `app.chain`) as a param so this function doesn't require access to app state